app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -491,4 +511,46 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
     public boolean isBackEnabled() {
         return false;
     }
-}
+    
+    private void populateAvailableAppsSpinner() {
+        Spinner spinner = (Spinner) findViewById(R.id.app_selection_spinner);
+        ArrayList<ApplicationRecord> readyApps = CommCareApplication._().getReadyAppRecords();

```

Comment:
But is it an intelligible order? Like, alphabetical is good, but if it's something like date of installation, that's not great.